### ISSUE TITLE
Date format incorrect?

### ISSUE THREAD

**Issue by krickler** [AirenSoft/OvenMediaEngine#640]
**Describe the bug**

I'm trying to use the API with a golang client, and it cant parse the dates the API server returns, it seems to me like the dates are incorrectly formatted.

This is what i get from the API server,
2022-01-17T15:07:58.325+0000 

This is what is described in the documentation:
2021-01-11T03:39:38.802+09:00

Difference is the last colon, third character from the right.

**To Reproduce**

Try to get stream statistics from API Server. 
/v1/stats/current/vhosts/default/apps/app/streams/${STREAM}

**Expected behavior**
Correctly formatted dates

**Logs**
No error in logs

**Server (please complete the following information):**
 - OS: Linux
 - OvenMediaEngine Version: v0.12.9

**Comment by basisbit** [comment]
a `:` there would be very much not standard-compliant / unusual as part of the timezone. Probably only the documentation needs an update. Where specifically did you find it with `:` in the documentation?

**Comment by dimiden** [comment]
The time expression form of OME follows [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) and [ISO8601](https://en.wikipedia.org/wiki/ISO_8601), and as shown in the document, it's intended that a ```:``` is included in the timezone part.
```
   date-fullyear   = 4DIGIT
   date-month      = 2DIGIT  ; 01-12
   date-mday       = 2DIGIT  ; 01-28, 01-29, 01-30, 01-31 based on
                             ; month/year
   time-hour       = 2DIGIT  ; 00-23
   time-minute     = 2DIGIT  ; 00-59
   time-second     = 2DIGIT  ; 00-58, 00-59, 00-60 based on leap second
                             ; rules
   time-secfrac    = "." 1*DIGIT
   time-numoffset  = ("+" / "-") time-hour ":" time-minute
   time-offset     = "Z" / time-numoffset

   partial-time    = time-hour ":" time-minute ":" time-second
                     [time-secfrac]
   full-date       = date-fullyear "-" date-month "-" date-mday
   full-time       = partial-time time-offset

   date-time       = full-date "T" full-time
```
Excluding a ```:``` from API results is a mistake, and I will change it. (I didn't know that ```std::put_time``` would return it in the form of ```+/-HHMM```.)

**Comment by dimiden** [comment]
@basisbit 
I think @krickler cited the response of [those APIs](https://app.gitbook.com/o/-Lcd-pxbm0NXHVOeV7t1/s/-Lcd00MyPtgKKaEeNUki-887967055/rest-api/v1/statistics/current#v1-stats-current-vhosts-vhost_name-apps-app_name-streams-stream)

**Comment by dimiden** [comment]
@krickler 
I solved this issue (7a80977a33f6cbd822883077a2fc345524b2787b). Thank you for the reporting!
